ansible/playbooks/openadk.yaml

```
@@ -0,0 +1,46 @@
+- name: Set up source development environments for Autoware Universe
+  hosts: localhost
+  connection: local
+  pre_tasks:
+    - name: Verify OS
+      ansible.builtin.fail:
+        msg: Only Ubuntu 22.04 is supported for this branch. Please refer to https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/.
+      when: ansible_distribution_version != '22.04'
```

This should also check that it's Ubuntu:

```suggestion
      when: ansible_distribution == 'Ubuntu' && ansible_distribution_version != '22.04'
```

---

@oguzkaganozt did you commit this change locally? I see it's marked as resolved but that distro is not checked

---

Changes now reflected into PR